playwright/src/main/java/com/microsoft/playwright/impl/BrowserImpl.java

```diff
@@ -65,17 +65,11 @@ public void close() {
   }
 
   private void closeImpl() {
-    if (isConnectedOverWebSocket) {
-      try {
-        connection.close();
-      } catch (IOException e) {
-        throw new PlaywrightException("Failed to close browser connection", e);
-      }
-      notifyRemoteClosed();
-      return;
-    }
     try {
       sendMessage("close");
```

Comment:
When we connected over web socket it's ok to simply close the connection, the server should handle that gracefully anyway (e.g. when the network connection is aborted). If this change fixes a particular bug please add a test (see [TestBrowserTypeConnect.java](https://github.com/microsoft/playwright-java/blob/master/playwright/src/test/java/com/microsoft/playwright/TestBrowserTypeConnect.java) for examples).